### ISSUE TITLE
Move placing the storage node into maintenance into the steps section

### ISSUE THREAD

**Issue by siegfried-h** [NetAppDocs/storagegrid-116#7]
Page: [Upgrade SANtricity OS on storage controllers using maintenance mode](https://docs.netapp.com/us-en/storagegrid-116/sg6000/upgrading-santricity-os-on-storage-controllers-using-maintenance-mode-sg6000.html)

This is identical to my other request. Please move the procedure to place a node into maintenance to the steps section:
"If the StorageGRID appliance is running in a StorageGRID system, the SG6000-CN controller has been placed in maintenance mode."

People are overseeing this and cause system downs and data loss.

regards
Siegfried Hepp

**Comment by netapp-madkat** [comment]
Hi Siegfried, 

Thank you for your suggestion. We've made the requested change and should be able to refresh the 11.6 doc site within the next few days.

Kathy

P.S. For consistency with other topics, we decided to leave maintenance mode as a prerequisite and to add a "confirm the appliance is in maintenance mode" step. Now that the maintenance mode requirement is mentioned in both places, it will be hard to miss.
